interactions/models/discord/auto_mod.py

```diff
@@ -369,7 +409,12 @@ def member(self) -> "Optional[Member]":
 TRIGGER_MAPPING = {
     AutoModTriggerType.KEYWORD: KeywordTrigger,
     AutoModTriggerType.HARMFUL_LINK: HarmfulLinkFilter,
+    AutoModTriggerType.SPAM: SpamTrigger,
     AutoModTriggerType.KEYWORD_PRESET: KeywordPresetTrigger,
     AutoModTriggerType.MENTION_SPAM: MentionSpamTrigger,
     AutoModTriggerType.MEMBER_PROFILE: MemberProfileTrigger,
 }
+
+TYPE_ALL_TRIGGER = Union[KeywordTrigger, SpamTrigger, KeywordPresetTrigger, MentionSpamTrigger, MemberProfileTrigger]
+
+TYPE_ALL_ACTION = Union[BlockMessage, AlertMessage, TimeoutUser, BlockMemberInteraction]
```

Comment:
You probably want to add these to `__all__`

Reply:
Sure, but to what level should they be added? Only in this file, or all `__init__`? I also just noticed that the Actions and Triggers are not added either in there...

Reply:
leave those for now, just add your new obects